app/controllers/meetings_controller.rb

```diff
@@ -33,7 +34,39 @@ def create
     redirect_unless_leader_for(@group) && return
 
     render :new unless @meeting.save
-    set_meeting_member
+    create_meeting_member
+  end
+
+  # POST /meetings/id/add_to_google_cal
```

Comment:
Nice work @yasirazgar,  a couple of refactoring suggestions:

1. You could refactor this path as `/meetings/id/[:action]/google_calender` where action can be `add` or `remove`
2. The methods `add_to_google_cal` and `remove_from_google_cal`could be combined into one method called google_calender to make it DRY.

Reply:
Hmmm, but I think that it would be nice if the both actions are handled into its own methods, and each of them have there own (single) responsibility.
If we maintain single method we may need to add conditional statements to differentiate between add/remove and soon it will get messy in the future if any additional features gets added to them.
Definitely your approach looks DRY, but I think we can sacrifice being DRY here for code clarity,
what do you think? please let me know your thoughts.

Reply:
Even better, I have moved the google_calendar methods into separate controller.